contracts/seed/Seed.sol

```diff
@@ -642,9 +642,9 @@ contract Seed {
             maximumReached || (minimumReached && block.timestamp >= endTime),
             "Seed: cannot withdraw while funding tokens can still be withdrawn by contributors"
         );
-        uint256 pendingFundingBalance = fundingCollected - fundingWithdrawn;
         fundingWithdrawn = fundingCollected;
-        fundingToken.safeTransfer(msg.sender, pendingFundingBalance);         
+        uint256 toWithdraw = fundingToken.balanceOf(address(this));
+        fundingToken.transfer(msg.sender, toWithdraw);
```

Comment:
we can do just `fundingToken.transfer(msg.sender, fundingToken.balanceOf(address(this));`

and add a comment on what's happening here for clarity